### PR TITLE
(fix) fixes to dependencies and webpack

### PR DESCRIPTION
**Pull request by pirupius**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "packages/*"
   ],
   "scripts": {
-    "start": "openmrs develop --sources 'packages/esm-*-app'",
+    "start": "openmrs develop --backend https://ugandaemr-backend.mets.or.ug --sources 'packages/esm-*-app'",
+    "start2": "openmrs develop --backend https://ohri-dev.globalhealthapp.net --sources 'packages/esm-*-app'",
```
Comment (slubwama) [inline package.json:10]: @pirupius  can we leave the default backend or point it to [ugandaemr](https://ugandaemr-backend.mets.or.ug/)

**Review by slubwama — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (slubwama) [inline package.json:10]: @pirupius why do we need start2

Comment (pirupius) [reply package.json:10]: Sorry about this, i added the second one cause the ugandaemr backend hadn't been updated but will remove this

Comment (pirupius) [reply package.json:10]: Same comment as above. `ugandaemr` backend hadn't been updated.

**Pull request merged**